classes/clean.php

```diff
@@ -122,12 +122,16 @@ static protected function update_status() {
             self::$tasks[$taskname] = time();
         }
 
-        printf("\r %-20s %4d%% (%d/%d)    $timeleft  \n", $taskname, $perc, $itemno, $total);
+        if (!isset(self::$options['verbose']) || self::$options['verbose'] == true) {
```

Comment:
Correct me if I'm wrong, won't this print out when the `verbose` option is not set or it's set to `true`? So by default, it will print out? Wouldn't it be better then to use a conditional more like: 
```isset(self::$options['verbose']) && self::$options['verbose'] == true```
This way it will only print when verbose is passed and true.

Reply:
In addition to my last, reading your comment above, this may be desired result, to default to displaying the print messages, if that's the case let me know and other than that it looks good.